src/diffusers/pipelines/pndm/pipeline_pndm.py

```diff
@@ -43,14 +43,14 @@ def __call__(self, batch_size=1, generator=None, torch_device=None, num_inferenc
         )
         image = image.to(torch_device)
 
-        prk_time_steps = self.scheduler.get_prk_time_steps(num_inference_steps)
+        prk_time_steps = self.scheduler.get_timesteps(num_inference_steps, step_mode="prk")
```

Comment:
Ideally I'd like to not have a `get_...` function at all, but just `self.prk_timesteps` and `self.plms_timesteps` - what do you think?